atomic_reactor/plugins/pre_resolve_composes.py

```diff
@@ -198,7 +243,15 @@ def request_compose_if_needed(self):
 
         compose_request = self.compose_config.render_request()
         compose_info = self.odcs_client.start_compose(**compose_request)
-        self.compose_ids = [compose_info['id'], ]
+        if self.compose_config.pulp:
+            self.composes_info = []
+            for arch in self.compose_config.arches:
+                compose_request = self.compose_config.render_pulp_request(arch)
+                compose_info = self.odcs_client.start_compose(**compose_request)
+                self.composes_info.append(compose_info)
+            self.compose_ids = [item['id'] for item in self.composes_info]
+        else:
```

Comment:
You can have `pulp` and `tag` type composes on the same build.

Reply:
fixed.